### PR TITLE
[JENKINS-67609] Deduplicate help button question mark

### PR DESCRIPTION
**Pull request by NotMyFault**
Fixes [JENKINS-67609](https://issues.jenkins.io/browse/JENKINS-67609)

The question mark on the new help button is set twice, but is only shown twice if you are using different themes.

The change proposed removes the additional question mark and sticks to the "major" one.

Looks like that was originally introduced in https://github.com/jenkinsci/jenkins/pull/5842.

### Proposed changelog entries

* Render the question mark on the new help button only once so that it is not shown twice, even while using different themes.

### Proposed upgrade guidelines

N/A

### Maintainer checklist

Before the changes are marked as `ready-for-merge`: 

- [ ] There are at least 2 approvals for the pull request and no outstanding requests for change
- [ ] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [ ] Changelog entries in the PR title and/or `Proposed changelog entries` are correct
- [ ] Proper changelog labels are set so that the changelog can be generated automatically
- [ ] If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387))
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins-ci.org/issues/?filter=12146)).

### CODE REVIEW

File: war/src/main/less/modules/form.less
Comment (timja) [inline war/src/main/less/modules/form.less:313]: hmm this might be intentional so that screen readers announce it, cc @janfaracik

Comment (janfaracik) [reply war/src/main/less/modules/form.less:313]: Hmm had a look at it and it seems all good to be removed 👍 Not too sure why I set the contents twice, good spot @NotMyFault. 

Would be good in the future to replace the link text with a Jenkins Symbol (eg ? in SVG form) so that the appearance of the link is consistent across browsers/fonts, and then adding some invisible text with `.jenkins-visually-hidden` to help screenreaders - although no need for that to be done in this PR.

Cheers :)

**Review by timja — APPROVED** [review]

**Review by res0nance — APPROVED** [review]

**Comment by timja** [comment]
This PR is now ready for merge, after ~24 hours, we will merge it if there's no negative feedback.

Thanks!

**Comment by daniel-beck** [comment]
Probably a viable LTS candidate for 2.332.x?

**Comment by NotMyFault** [comment]
> Probably a viable LTS candidate for 2.332.x?

My Jira issue is [here](https://issues.jenkins.io/browse/JENKINS-67609), in case you want to label it as lts candidate.
But worth to cover I guess, considering it decreases the readability on different themes a lot.

**Comment by daniel-beck** [comment]
@NotMyFault Thanks!

**Pull request merged**
